src/sagemaker_pytorch_serving_container/etc/log4j2.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+    <Appenders>
+        <Console name="STDOUT" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{ISO8601} [%-5p] %t %c - %m%n"/>
+        </Console>
+        <RollingFile
+                name="access_log"
+                fileName="${env:LOG_LOCATION:-logs}/access_log.log"
+                filePattern="${env:LOG_LOCATION:-logs}/access_log.%d{dd-MMM}.log.gz">
+            <PatternLayout pattern="%d{ISO8601} - %m%n"/>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="10 MB"/>
+                <TimeBasedTriggeringPolicy/>
+            </Policies>
+            <DefaultRolloverStrategy max="5"/>
+        </RollingFile>
+        <RollingFile
+                name="model_log"
+                fileName="${env:LOG_LOCATION:-logs}/model_log.log"
+                filePattern="${env:LOG_LOCATION:-logs}/model_log.%d{dd-MMM}.log.gz">
+            <PatternLayout pattern="%d{ISO8601} [%-5p] %c - %m%n"/>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="10 MB"/>
+                <TimeBasedTriggeringPolicy/>
+            </Policies>
+            <DefaultRolloverStrategy max="5"/>
+        </RollingFile>
+        <RollingFile
+                name="model_metrics"
+                fileName="${env:METRICS_LOCATION:-logs}/model_metrics.log"
+                filePattern="${env:METRICS_LOCATION:-logs}/model_metrics.%d{dd-MMM}.log.gz">
+            <PatternLayout pattern="%d{ISO8601} - %m%n"/>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="10 MB"/>
+                <TimeBasedTriggeringPolicy/>
+            </Policies>
+            <DefaultRolloverStrategy max="5"/>
+        </RollingFile>
+        <RollingFile
+                name="ts_log"
+                fileName="${env:LOG_LOCATION:-logs}/ts_log.log"
+                filePattern="${env:LOG_LOCATION:-logs}/ts_log.%d{dd-MMM}.log.gz">
+            <PatternLayout pattern="%d{ISO8601} [%-5p] %t %c - %m%n"/>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="10 MB"/>
+                <TimeBasedTriggeringPolicy/>
+            </Policies>
+            <DefaultRolloverStrategy max="5"/>
+        </RollingFile>
+        <RollingFile
+                name="ts_metrics"
+                fileName="${env:METRICS_LOCATION:-logs}/ts_metrics.log"
+                filePattern="${env:METRICS_LOCATION:-logs}/ts_metrics.%d{dd-MMM}.log.gz">
+            <PatternLayout pattern="%d{ISO8601} - %m%n"/>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="10 MB"/>
+                <TimeBasedTriggeringPolicy/>
+            </Policies>
+            <DefaultRolloverStrategy max="5"/>
+        </RollingFile>
+    </Appenders>
+    <Loggers>
+        <Logger name="ACCESS_LOG" level="info">
+            <AppenderRef ref="access_log"/>
+        </Logger>
+        <Logger name="io.netty" level="error"/>
+        <Logger name="MODEL_LOG" level="warn">
```

Comment:
MODEL_LOG level is set differently in new config (was INFO, now is WARN)

Reply:
Thanks, fixed.